src/test/java/Homework20.java

```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+public class Homework20 extends BaseTest {
+    String nameNewPlaylist = "AutomationGuruGroove";
+
+    @Test
+    public void createAndDeletePlayListTest() throws InterruptedException {
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.